src/AutoWrapper/Base/WrapperBase.cs

```diff
@@ -50,6 +52,9 @@ public virtual async Task InvokeAsyncBase(HttpContext context, AutoWrapperMember
 
                     if (context.Response.HasStarted) { LogResponseHasStartedError(); return; }
 
+                    var url = Microsoft.AspNetCore.Http.Extensions.UriHelper.BuildRelative(context.Request.Path);
```

Comment:
There seems to be no call to it, so consider cleaning it up.

Reply:
ok, cleaned up.